### PR TITLE
Keithkurson/ch4156/update packages in community

### PR DESCRIPTION
**Pull request by keithk**
## Links
* Remix link: https://quilled-helium.glitch.me/
* Issue-tracker link: https://app.clubhouse.io/glitch/story/4156/update-packages-in-community

## Changes:
* Updated order of packages by ABC
* Updates the package.json to the newest *minor* version for all of our packages

## How To Test:
* Pull down branch and run

## Feedback I'm looking for:

### CODE REVIEW

**Comment by keithk** [comment]
Taking a look at some things that @clottman flagged 🎏

File: src/components/collection/collection-item-small.js [hunk 1]
```diff
@@ -44,7 +44,7 @@ const CollectionItemSmall = ({ collection, showCurator }) => (
         <div className={styles.nameArea}>
           <div className={styles.collectionNameWrap}>
             <div className={styles.itemButtonWrap}>
-              <Button textWrap as="span" image={collection.private ? <Icon icon="private" /> : null} imagePosition="left">
+              <Button textwrap as="span" image={collection.private ? <Icon icon="private" /> : null} imagePosition="left">
```
Comment (keithk) [inline src/components/collection/collection-item-small.js]: Was getting a lot of warnings that textWrap didn't exist and needed to be `textwrap`, so I just updated all of them in this PR too.

Comment (clottman) [reply src/components/collection/collection-item-small.js]: Hmm, that's weird! The shared component source definitely specifies `textWrap`. Are you on the latest version of shared-components?

Comment (clottman) [reply src/components/collection/collection-item-small.js]: https://glitch.com/edit/#!/shared-components?path=lib/button.js:56:12

Comment (sarahzinger) [reply src/components/collection/collection-item-small.js]: Yeah btw I'm still getting an error about textwrap on the homepage of your branch:
Screen Shot 2020-01-02 at 4 44 57 PM

Comment (sarahzinger) [reply src/components/collection/collection-item-small.js]: oh you know what I'm getting these errors on master too, so i guess nevermind lol

**Pull request merged**